### PR TITLE
Mobile active > add a note: attach photo | Implementation

### PR DESCRIPTION
**Pull request by LinX64**
Dialog, translations and permissions check for the add note feature.

- No need for rebasing with this PR as I am creating new branches from here.

### CODE REVIEW

**Comment by LinX64** [comment]
[Mobile active > add a note: attach photo](https://trello.com/c/TL4MVMzM/1174-mobile-active-add-a-note-attach-photo)

**Comment by LinX64** [comment]
[Attach Note: Helper Dialog - implementation for permission](https://trello.com/c/0zBbKQpm/1731-attach-note-helper-dialog-implementation-for-permission)

File: app/src/main/java/pl/llp/aircasting/ui/view/screens/session_view/map/MapController.kt [hunk 1]
```diff
@@ -26,17 +28,18 @@ open class MapController(
     mViewMvc: SessionDetailsViewMvc?,
     sessionUUID: String,
     sensorName: String?,
-    fragmentManager: FragmentManager,
+    val mFragmentManager: FragmentManager,
```
Comment (xameYeoh) [inline app/src/main/java/pl/llp/aircasting/ui/view/screens/session_view/map/MapController.kt:31]: I think this could be private

**Review by xameYeoh — APPROVED** [review]
Looks very neat! 😄 
I've left some minor comments

File: app/src/main/java/pl/llp/aircasting/ui/view/screens/session_view/map/MapController.kt (hunk 1 above)
Comment (LinX64) [reply app/src/main/java/pl/llp/aircasting/ui/view/screens/session_view/map/MapController.kt:31]: Not exactly; check the SessionDetailsViewController() and it's constructor.

**Pull request closed without merging**